src/styles/animation.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
```

Comment:
Consider using more descriptive class names instead of `.item-enter-active`. For example, `.accordion-item-enter-active`. 🎨